src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -785,6 +785,20 @@ export const getAppletTabs = ({
   },
 ];
 
+export const testIsReportCommonFieldsRequired = (
+  isScoreReport: boolean,
+  value: boolean,
+  context: unknown,
+) => {
+  if (isScoreReport) {
+    const conditionalLogic = get(context, 'from.0.value.conditionalLogic')?.length;
+
+    return !!conditionalLogic || value;
```

Comment:
is the double exclamation mark needed here?
as I see above, conditional logic can be undefined, 0, or other number

Reply:
this function should return boolean. so !! is required